src/coreclr/jit/fgprofile.cpp

```diff
@@ -47,6 +47,31 @@ bool Compiler::fgHaveProfileData()
     return (fgPgoSchema != nullptr);
 }
 
+//------------------------------------------------------------------------
+// fgHaveSufficientProfileData: check if profile data is available
+//   and is sufficient enough to be trustful.
+//
+// Returns:
+//   true if so
+//
+// Note:
+//   See notes for fgHaveProfileData.
+//
+bool Compiler::fgHaveSufficientProfileData()
+{
+    if (!fgHaveProfileData())
+    {
+        return false;
+    }
+
+    if ((fgFirstBB != nullptr) && (fgPgoSource == ICorJitInfo::PgoSource::Static))
+    {
+        const BasicBlock::weight_t sufficientSamples = 5000;
```

Comment:
This used to be 1000 -- I assume you're increasing this to keep prejit image size small?

If so, you should add a comment describing how this value influences prejit size.
If not, you might comment on what the impact of changing this would be.

Reply:
1000 samples feels like a plenty of evidence that something is hot.

Reply:
Reverted to 1000. Yes, this changes was supposed to decrease the prejitted size, I was using this histogram:
![image](https://user-images.githubusercontent.com/523221/125352203-a25ce400-e369-11eb-8bd3-014869dca90f.png)
(weights in SPC).

However, I don't need to save some space with it anymore as I've found an unrelated issue that bloated size for no reason (binary expressions like "arg op cns" used to leave "cns" on top of the pushed stack so there were lots of false-positive foldable-branches/switches).